### DISCUSSION TITLE
Jpeg, Jpg and png Images are not returning same format as .svg image from api call.

### DISCUSSION THREAD

**Discussion by pchebro** [directus/directus#8672] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

Hi, 
I'm not sure this is even a bug but need help understanding how the image response will be from Directus api irrespective of the image type. svg images are loading fine on angular UI with below response but any other image formats are not rendering.

**Below is the response format I can see for .svg image**
![image](https://user-images.githubusercontent.com/89423003/136609317-69739a84-6055-4740-a256-88c7cf167d15.png)

**Below is the response format for .jpeg or .png**

![image](https://user-images.githubusercontent.com/89423003/136609395-946011f9-4d80-48a1-9872-8d16b7ec2802.png)

### To Reproduce

You can use any existing jpeg or png image in directus and try to render that in Angular UI. 

### What version of Directus are you using?

9.0.0-rc.64

### What version of Node.js are you using?

12.11.1

### What database are you using?

MySQL

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10

### How are you deploying Directus?

PaaS

**Comment by rijkvanzanten** [comment]
@pchebro SVGs are XML-based text documents (in utf-8), PNG/JPEG/ a ton of other file formats are proprietary binary based file payloads. They're different by design, this is not something done in/by Directus 👍🏻
